Mathlib/CategoryTheory/Monoidal/Mod_.lean

```diff
@@ -129,3 +128,47 @@ def comap {A B : Mon_ C} (f : A ⟶ B) : Mod_ B ⥤ Mod_ A where
 -- Lots more could be said about `comap`, e.g. how it interacts with
 -- identities, compositions, and equalities of monoid object morphisms.
 end Mod_
+
+section Mod_Class
+
+open CategoryTheory Mon_Class MonoidalCategory
+
+variable (M : C) [Mon_Class M]
+
+/-- An action of a monoid object `M` on an object `X` is the data of a map `smul : M ⊗ X ⟶ X` that
+satisfies unitality and associativity with multiplication.
+
+See `MulAction` for the non-categorical version. -/
+class Mod_Class (X : C) where
+  /-- The action map -/
+  smul : M ⊗ X ⟶ X
```

Comment:
In the bundled version, this field was denoted `act`, `one_smul` was `one_act` and `mul_smul` was `assoc`. I think it would be better to have consistent names.

Reply:
The hope was to change the names of `Mod_` to match `Mod_Class`, rather than the other way around